packages/lib/intervalLimits/server/getBusyTimesFromLimits.ts

```diff
@@ -284,3 +291,84 @@ const _getBusyTimesFromTeamLimits = async (
 
   return limitManager.getBusyTimes();
 };
+
+export const getBusyTimesFromGlobalBookingLimits = async (
+  ...args: Parameters<typeof _getBusyTimesFromGlobalBookingLimits>
+): Promise<ReturnType<typeof _getBusyTimesFromGlobalBookingLimits>> => {
+  return monitorCallbackAsync(_getBusyTimesFromGlobalBookingLimits, ...args);
+};
+
+const _getBusyTimesFromGlobalBookingLimits = async (
+  userId: number,
+  userEmail: string,
+  bookingLimits: IntervalLimit,
+  dateFrom: Dayjs,
+  dateTo: Dayjs,
+  rescheduleUid?: string
+) => {
+  const { limitDateFrom, limitDateTo } = getStartEndDateforLimitCheck(
+    dateFrom.toISOString(),
+    dateTo.toISOString(),
+    bookingLimits
+  );
+
+  const where: Prisma.BookingWhereInput = {
+    userId,
+    status: BookingStatus.ACCEPTED,
+    startTime: {
+      gte: limitDateFrom.toDate(),
+    },
+    endTime: {
+      lte: limitDateTo.toDate(),
+    },
+    eventType: {
+      schedulingType: null,
+    },
+  };
+
+  if (rescheduleUid) {
+    where.NOT = {
+      uid: rescheduleUid,
+    };
+  }
+
+  const bookings = await prisma.booking.findMany({
+    where,
+    select: {
+      id: true,
+      startTime: true,
+      endTime: true,
+      eventType: {
+        select: {
+          id: true,
+        },
+      },
+      title: true,
+      userId: true,
+    },
+  });
+
+  const busyTimes = bookings.map(({ id, startTime, endTime, eventType, title, userId }) => ({
+    start: dayjs(startTime).toDate(),
+    end: dayjs(endTime).toDate(),
+    title,
+    source: `eventType-${eventType?.id}-booking-${id}`,
+    userId,
+  }));
+
+  const limitManager = new LimitManager();
+
+  await getBusyTimesFromBookingLimits({
```

Comment:
I think we need to pass the organizer's timezone here. If not can you please explain why 🙏

Reply:
It was not intentional, i had missed it somehow, fixed now!